force-app/main/fslStatusTransition/classes/HOT_WorkOrderStatusHandler.cls

```diff
@@ -74,6 +82,12 @@ public without sharing class HOT_WorkOrderStatusHandler extends MyTriggers {
         update requestsToUpdate;
     }
 
+    private static void updateStatusChangedFieldOnWorkOrder(List<WorkOrder> changedWorkOrders) {
+        for (WorkOrder workOrder : changedWorkOrders) {
+            workOrder.HOT_SendSMSNotificationOnStatusChanged__c = true;
+        }
+    }
+
     private static void updateWorkOrderStatus(List<WorkOrder> workOrders) {
         List<WorkOrderLineItem> workOrderLineItems = [
             SELECT WorkOrderId, Status
```

Comment:
Denne vil vel huke av for alle statusendringer.. Er det ønskelig? Selv om Batch-en bare filtrerer ut på status... Navnet på feltet tilsier jo at det skal sendes sms når det er huket av.